resources/definitions/3DI_D300_v2

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
I can't find any "v2" of this printer. Is it just v2 of the profile, or is there a specification page about this printer online?

Reply:
The version was stopped. Right now we are making the version 2 while adding this to Cura

Reply:
You can open a new PR once the requested changes are made, then.